### PR TITLE
ADBDEV-2916 add astralinux support

### PR DESCRIPTION
**Pull request by deart2k**
## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

**Comment by deart2k** [comment]
@Stolb27 Result of build with "-Werror=implicit-fallthrough=3"
https://ci.arenadata.io/view/Projects/job/adb-6.x-build-project/1004/console

File: configure [hunk 1]
```diff
@@ -5495,7 +5495,7 @@ fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $pgac_cv_prog_cc_cflags__Werror_implicit_fallthrough_3" >&5
 $as_echo "$pgac_cv_prog_cc_cflags__Werror_implicit_fallthrough_3" >&6; }
 if test x"$pgac_cv_prog_cc_cflags__Werror_implicit_fallthrough_3" = x"yes"; then
-  CFLAGS="$CFLAGS -Werror=implicit-fallthrough=3"
```
Comment (Stolb27) [inline configure]: I've built successfully gpdb in the astra container without these changes. Can you describe motivation?

**Comment by Stolb27** [comment]
There are a few more changes in the corresponding commit for Alt linux: https://github.com/arenadata/gpdb/commit/0443d47817ef8f8a9006c75f543daa27a332d01a. I think we need the similar changes in:

1. gpAux/Makefile.global
2. concourse/scripts/common.bash

File: configure (hunk 1 above)
Comment (deart2k) [reply configure]: This changes was removed

**Comment by Stolb27** [comment]
@vitaliy-popov, please make line indentation consistent with modified files (spaces vs tabs)

File: gpAux/releng/set_bld_arch.sh [hunk 2]
```diff
@@ -11,6 +11,14 @@ case "`uname -s`" in
             ;;
         esac
     fi
+
+    if [ -f /etc/astra_version ]; then
+	case "`cat /etc/astra_version`" in
+		*)
```
Comment (Stolb27) [inline gpAux/releng/set_bld_arch.sh]: Do we really need to copy this useless `case` statement?

**Review by Stolb27 — APPROVED** [review]

**Pull request merged**